app/controllers/admin/enterprises_controller.rb

```diff
@@ -114,8 +114,36 @@ def visible
       end
     end
 
+    def invite_manager
```

Comment:
Can we please rethink this? We should have a new Rest resource and so have a regular rest controller IMO. Following this convention would save us some work.

Reply:
I'd rather discuss this design before jumping into an implementation.

Reply:
I wasn't sure exactly where to put it. We're calling `EnterpriseMailer`, and adding a `User` to an `Enterprise`, (via an `EnterpriseRole`), so I put it in the `EnterpriseController`. Would you rather have the logic in a new `EnterpriseRoleController`?

Reply:
Maybe `invite_enterprise_user` would be a better method name than `invite_manager`?

Reply:
My point was more about (in terms of routes):

```ruby
resources :enterprises do
  resource :invitations, only: [:create]
end
```